web-api/src/storage/postgres.rs

```diff
@@ -322,7 +318,9 @@ impl Database {
             last_view
         ) ",
         );
-        for chunk in cois.chunks(Database::BIND_LIMIT / 7) {
```

Comment:
:exclamation: : Itertools `chunks` extension is not compatible with `await` (it's not `Send`).